js/src/components/dismissible-notice/index.js

```diff
@@ -16,26 +17,30 @@ import localStorage from '.~/utils/localStorage';
  *
  * @param {Object} props React props.
  * @param {JSX.Element} props.children Children to render.
- * @param {string} props.className Class to use in the Notice component.
  * @param {string|null} props.localStorageKey Local Storage Key where is keep the dismiss state.
- * @return {JSX.Element} {@link Notice} element with the info message and the link to the documentation.
+ * @param {Function} props.onRemove Callback when clicking on remove notice
+ * @param {Object} props.restProps Props to be forwarded to Notice Component. Like className.
+ * @return {JSX.Element} {@link Notice} Dismissible Notice element
  */
 const DismissibleNotice = ( {
 	children,
-	className,
 	localStorageKey = null,
+	onRemove = noop,
+	...rest
 } ) => {
 	const defaultDismissedValue = localStorageKey
-		? !! localStorage.get( localStorageKey )
+		? localStorage.get( localStorageKey ) === 'true'
 		: false;
```

Comment:
💅 

This could be simplified as 

const defaultDismissedValue = localStorage.get( localStorageKey ) === 'true';